ChangeLog.d/sha3.txt

```diff
@@ -0,0 +1,3 @@
+Features
+    * Add SHA3 family hash functions.
```

Comment:
Minor: please be specific about which variants have been added. Please write SHA3 as SHA-3 (lots of instances of this).